internal/lib/lib.go

```diff
@@ -457,6 +457,15 @@ func GetGlobalBgpPeerLabels() []string {
 	return bgpPeerLabels
 }
 
+func GetT1LRPath() string {
+	t1LrPath := os.Getenv("NSXT_T1_LR")
+	if t1LrPath == "" {
+		utils.AviLog.Error("Required param nsxt1LR, missing from the configuration")
```

Comment:
Do we need the error log here? This might lead to wrong error messages, as it suggests the field to be required?
For instance, in the translator and the rest layer code where we don't have the cloud type check, and we just call this function, with a non NSX-T cloud it would still print this error.

Reply:
Good catch. Let me remove the log from here.